content/docs/operating/security.md

```diff
@@ -161,3 +161,16 @@ members of the Prometheus development team and the staff of those providers
 have access. If you are concerned about the exact provenance of your binaries,
 it is recommended to build them yourself rather than relying on the
 pre-built binaries provided by the project.
+
+## External audits
+
+[CNCF](https://cncf.io) sponsored an external security audit by
+[cure53](https://cure53.de) which ran from April 2018 to June 2018.
+
+The audit found concerns regarding Prometheus' secrity model, but following
+what is layed out in this document should guard against most of them.
+In particular, the CORS (PRM-01-001) and CSRF (PRM-01-003) attack vectors might
+be non-obvious.
+
+For more details, please read the
+[final report of the audit](assets/downloads/2018-06-11--cure53_security_audit.pdf).
```

Comment:
Remove the `assets` from this URL

Reply:
Are you certain? `grep -ri assets content` implies otherwise.

Reply:
Okay, I see now that there's a URL re-write rule in place for static assets. I do think, though, that the URL needs to begin with `/assets` rather than `assets` since this is not the index page.